src/rating/emoticon-rating.js

```diff
@@ -71,6 +71,7 @@ class EmoticonRating extends React.Component<
           tabIndex={isFocusable ? '0' : '-1'}
           aria-setsize={5}
           aria-checked={x === value}
+          size={size}
```

Comment:
to follow the shared prop naming convention, this prop should start with the $ prefix, so it should be `$size={size}`